diff --git a/io.github.microgamercz.piki.desktop b/io.github.microgamercz.piki.desktop
index 6d1ac2c..3c34ef0 100644
--- a/io.github.microgamercz.piki.desktop
+++ b/io.github.microgamercz.piki.desktop
@@ -6,7 +6,7 @@ Name=Piki
 Comment=Unofficial Kirigami client for Pixiv
 Comment[cs]=Neoficiální Kirigami klient pro pixiv
 Keywords=pixiv;anime;art;otaku;
-Version=0.4.0
+Version=1.0
 Exec=piki
 Icon=io.github.microgamercz.piki
 Type=Application
